app/models/strategy.rb

```diff
@@ -55,4 +55,8 @@ def published?
   def comments
     Comment.comments_from(self)
   end
+
+  def comments
```

Comment:
Looks like this is a duplicate of what exists a few lines above. Let's not include this change in this PR.